pyspecProcScripts/correlation_alignment.py

```diff
@@ -199,16 +214,17 @@ def correl_align(
             )
         else:
             f_shift = correl.run(np.real).argmax(direct)
-        s_copy = s_orig.C
-        s_copy *= np.exp(-1j * 2 * np.pi * f_shift * s_copy.fromaxis(direct))
-        s_orig.ft(direct)
-        s_copy.ft(direct)
+        # Calculate energy with shift applied
+        for_Ecalc = s_jk * np.exp(
+            -1j * 2 * np.pi * f_shift * s_jk.fromaxis(direct)
+        )
+        for_Ecalc.ft(direct)
         if my_iter == 0:
             logging.debug(psd.strm("holder"))
             if fl:
-                s_copy.reorder([direct], first=False)
+                for_Ecalc.reorder([direct], first=False)
```

Comment:
I do not like this `for_Ecalc` .  It's actually just the aligned signal --> maybe `s_aligned`.  Also, you should be starting form `s_orig` here, in keeping with comments above.  It's possible that at the beginning `s_orig` gets an ift, to be ft'd again before the end *without being modified*.

Reply:
I'm assuming you mean changing `for_Ecalc = s_jk * np.exp(-1j * 2 * np.pi * f_shift * s_jk.fromaxis(direct)`) to `for_Ecalc = s_orig * np.exp(-1j * 2 * np.pi * f_shift * s_jk.fromaxis(direct))`. But:

1.  see the strm below where it says "signal energy per transient (recalc to check that it stays the same):"..." which means I don't think we want to use s_orig here. 
2. doing so causes the example to fail / result isn't aligned

Reply:
but if you are talking about the dignostic, that runs still so I am going to go with that?